apps/src/netsim/NetSimMessage.js

```diff
@@ -69,8 +69,8 @@ NetSimMessage.send = function (shard, fromNodeID, toNodeID, payload, onComplete)
   entity.fromNodeID = fromNodeID;
   entity.toNodeID = toNodeID;
   entity.payload = payload;
-  entity.getTable_().create(entity.buildRow_(), function (row) {
-    onComplete(row !== undefined);
+  entity.getTable_().create(entity.buildRow_(), function (err) {
+    onComplete(err === null);
```

Comment:
I see there are a few more of these, so I'm going to stop commenting on each individually :)